tsconfig.base.json

```
@@ -14,7 +14,10 @@
     "skipLibCheck": true,
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
-    "paths": {}
+    "paths": {
+      "@data-access": ["libs/data-access/src/index.ts"],
+      "@web/*": ["apps/site/src/*"]
+    }
```

```suggestion
    "paths": {
      "@myflownfts/data-access": ["libs/data-access/src/index.ts"],
      "@myflownfts/site/*": ["apps/site/src/*"]
    }
```

The convention is to use same names as the directories to minimize confusion.